pkg/util/fwatch/watcher.go

```diff
@@ -26,16 +26,11 @@ import (
 	"os"
 	"sync"
 
-	"k8s.io/utils/inotify"
+	"github.com/fsnotify/fsnotify"
 
 	"github.com/arangodb/kube-arangodb/pkg/logging"
 )
 
-const (
-	inotifyEventsModification = inotify.InCreate | inotify.InDelete | inotify.InDeleteSelf |
```

Comment:
This is now not so optimal - there will be many events fired which does not mean that file content was actually changed.

But I guess this it is the price for cross-platform build, so LGTM.

Reply:
This one should fix it:
```
if event.Op == fsnotify.Chmod {
	continue
}
```

Reply:
@jwierzbo yeah, cool! Can you add that? feel free to merge afterwards